manifests/metallb.yaml

```diff
@@ -319,6 +321,11 @@ spec:
         ports:
         - containerPort: 7472
           name: monitoring
+        - containerPort: 7946
```

Comment:
Can you also update the comment about 20 lines above this that shows how to set `METALLB_ML_BIND_PORT` to note that if that port is changed, it should also be changed in these new places you're adding the port number?

Otherwise, this change lgtm

Reply:
Done